cmd/2019_5/main.go

```diff
@@ -27,11 +27,13 @@
 
 	var (
 		instructionsAC = read.Read(flagInput, separator)
+		chanAC, chanTR = make(chan int, 1), make(chan int, 1)
 		instructionsTR = make([]int, len(instructionsAC))
 	)
 
 	copy(instructionsTR, instructionsAC)
-
-	utils.AssertEqual(intcode.Compute(instructionsAC, intcode.Output, inputAC), resComputeAC)
-	utils.AssertEqual(intcode.Compute(instructionsTR, intcode.Equals, inputTR), resComputeTR)
+	chanAC <- inputAC
+	chanTR <- inputTR
+	utils.AssertEqual(intcode.Compute(instructionsAC, intcode.Output, chanAC), resComputeAC)
+	utils.AssertEqual(intcode.Compute(instructionsTR, intcode.Equals, chanTR), resComputeTR)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 92 characters, out of limit 80](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)